packages/compiler/src/transforms/add-imports.ts

```diff
@@ -15,29 +15,80 @@ import type { Transformer } from '../transform';
 import type { Logger } from '@openfn/logger';
 
 const globals = [
+  'AggregateError',
+  'Array',
+  'ArrayBuffer',
+  'AsyncFuction',
+  'AsyncGenerator',
   'atob',
+  'Atomics',
+  'BigInt',
+  'BigInt64Array',
+  'BigUint64Array',
   'Blob',
+  'Boolean',
   'btoa',
   'Buffer',
   'clearInterval',
   'clearTimeout',
   'console',
+  'DataView',
   'Date',
+  'decodeURI',
+  'decodeURIComponent',
+  'encodeURI',
+  'encodeURIComponent',
   'Error',
+  'eval',
```

Comment:
I'm a little concerned about adding `eval` here, obviously I don't want to break any adaptors - but if we allow `eval` we have to ensure it is sufficiently isolated or proxied. Like that vm2 vuln. where the calling context is available via the Error prototype; if you think we should allow it and handle the sandboxing in a more holistic manner in the near future I'm ok with that.

Reply:
I'll push a commit in a few minutes with some unit tests, but globals like `Set` are available, while `eval` is disabled and `process` not in scope.

I have raised an issue #104  to review and improve unit testing around this.

Reply:
Ok - this is good to go! See the latest commit for a couple of comforting unit tests.

Re release - I suggest we merge this  into main, then release from the `release-next` branch and do it all in one go.

Reply:
.. apart from the failing test of course

Reply:
Ok, all done!